src/javascripts/helpers/viewHelper.js

```diff
@@ -0,0 +1,28 @@
+// import firebase from 'firebase/app';
```

Comment:
If you are not using it, remove it from production code please. No commented code.

Reply:
will do!